blog/templates/blog/blogbase.html

```diff
@@ -71,19 +71,19 @@
             <li><a href="#">One more separated link</a></li>
           </ul>
         </li>
-      </ul>
+      </ul> -->
       <form class="navbar-form navbar-left" role="search">
         <div class="form-group">
-          <input type="text" class="form-control" placeholder="Search">
+          <input type="text" name="tag" class="form-control" placeholder="Search">
         </div>
         <button type="submit" class="btn btn-default">Submit</button>
       </form>
       <ul class="nav navbar-nav navbar-right">
-        <li><a href="#">Link</a></li>
+        {% if user.is_authenticated %}<li>{{ user.get_full_name }}</li>{% endif %}
         <li class="dropdown">
           <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Dropdown <span class="caret"></span></a>
           <ul class="dropdown-menu">
-            <li><a href="#">Action</a></li>
+            
             <li><a href="#">Another action</a></li>
```

Comment:
Make these actual relevant drop down items or remove them.  No need to keep placeholders here.  Some good links here would be a logout link if a user is authenticated.